examples/dgfip_c/ml_primitif/common.ml

```diff
@@ -266,20 +268,36 @@ module TGV = struct
         set_bool tgv signvar (v < 0.0);
         set tgv dvar (Float.abs v)
 
-  let internal_copy ~ignore_undefined (tgv : t) var_list =
+  let reset_saisie_calc ~except tgv =
+    let save = get_map_opt tgv except in
+    reset_saisie_calculee tgv;
+    set_map tgv save
+
+  type undef_action =
```

Comment:
If I understand correctly this is a sum type specifying the 3 possible intended behavior when we copy a TGV and read from an undefined value, is it not ?

Reply:
It is. ;)